airflow/providers/amazon/aws/utils/emailer.py

```diff
@@ -36,8 +37,10 @@ def send_email(
 ) -> None:
     """Email backend for SES."""
     hook = SESHook(aws_conn_id=conn_id)
+    
+    mail_from = conf.get('smtp', 'SMTP_MAIL_FROM')
```

Comment:
Please guide if this is right approach or should we update the EmailOperator to have input for "from".

Reply:
It looks good. Can you add tests to avoid regression?

Reply:
Thanks @mik-laj, I will update the Pr with tests.

Reply:
@satishmittal111  Are you able to update this PR with tests?